app/styles/app.scss

```diff
@@ -1,14 +1,31 @@
-@import "nypr-ui";
-@import "nypr-ui/brand-header";
+// This is just for Bryn, take out later
```

Comment:
later is now

Reply:
but then you wouldn't have all the cool styles!!